### DISCUSSION TITLE
高通 X elite 處理器

### DISCUSSION THREAD

**Discussion by uwje** [1101-GS4719-C/Python-Community-GS4719-C#1682] [General]
高通在日前發布全新設計的X elite 電腦處理器。
在這之前，高通是以手機處理器而為人所知，在apple推出基於arm架構的處理器後，便推出基於arm架構的處理器，但結果卻差強人意，但在2021年左右吸收癌自apple晶片設計的人才後，在近日推出arm架構的新型晶片，整合了cpu gpu apu ，而與前代最大差異在於cou不再是arm的公版架構，而是自研發的架構，而apu則提供在影音處理上的加速，使其更流暢，也有更低的功耗，似乎能和apple的處理器做比較。
我認為這次高通發布的處理器的確在跑分上與其他處理器拉開差距，但實際應用上還是要看軟硬體的整合是否完美，而各大公司是否願意為了這個晶片而去重新編寫或改寫原本基於其他架構的軟體，也希望各大公司可以盡快跟上支持arm架構的軟體
參考資料:https://www.techbang.com/posts/110608-qualcomm-snapdragon-x-elite-processor-released-supports-win12

**Comment by Terryyyyyyyyyyyy** [comment]
我認為的確要有更多處理器公司投入發展基於arm架構的晶片，才會使軟體的廠商快速地發表基於arm架構軟體的版本，要不然要現在蘋果的做法僅是將他轉譯，會造成一定程度上的損耗，也希望能有遊戲廠商投入開發arm版的遊戲，使我們看到蘋果筆電在運作遊戲的效能如何。

**Comment by davidteng00000** [comment]
高通的X Elite處理器呈現嶄新的技術突破，結合了CPU、GPU和APU，顯著提升性能表現。然而，實際應用仍需看軟硬體整合，以及各公司對ARM架構的軟體支持。期待廠商迅速適應並支援這一新技術。

**Comment by mrweixd** [comment]
看到高通推出全新的X Elite電腦處理器，感到相當驚訝。過去他們主要以手機處理器聞名，但現在進入電腦領域，尤其是推出基於自家研發架構的處理器，顯示出高通在技術創新上的決心。雖然跑分上拉開了差距，但我認為實際應用還需考慮整合和相容性，期待軟硬體廠商迅速跟進，為這項新技術提供更多支援。

**Comment by eric202501** [comment]
我認為還需要考慮整合性能以及軟硬體的協同作用，高通需要獲得廣泛支持，才能讓各公司重新編寫或改寫被使用的架構，這對發揮X Elite的潛力來說很重要。
